mlir/test/Target/LLVMIR/omptarget-fortran-allocatable-types-host.mlir

```diff
@@ -0,0 +1,349 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
```

Comment:
Can these tests be minimized?

Reply:
I will try to hand reduce it as this is currently the output after running CSE/dce via mlir-opt, it's likely a little unnecessary for it to be a fully functional test case for the purpose of the IR checks in this test so I'll see what I can do.